pkg/ccl/backupccl/backup_destination.go

```diff
@@ -136,7 +137,45 @@ func resolveDest(
 	if err != nil {
 		return "", "", "", nil, nil, err
 	}
-	if !exists {
+	if exists && !dest.Exists && chosenSuffix != "" {
```

Comment:
could include a `IsActive(Start22_1)` here to avoid spurious errors in the mixed version state

Reply:
Done.